src/components/custom/_signup-affiliate.js

```diff
@@ -0,0 +1,189 @@
+
+    @media ${device.mobileL} {
+        min-height: auto;
+        padding: 13px 16px;
+    }
+`
+const InputGroup = styled.div`
+    position: relative;
+    width: 100%;
+    margin: 2.5rem 0 0.6rem;
+
+    @media ${device.mobileL} {
+        margin: 25px 0 16px 0;
+    }
+`
+const EmailButton = styled(Button)`
+    width: 100%;
+    font-size: 1.4rem;
+    margin: 3.2rem auto 0.4rem;
+
+    @media ${device.tabletL} {
+        margin-top: 24px;
+    }
+
+    @media ${device.mobileL} {
+        font-size: 12px;
+    }
+`
+const LoginText = styled(Header)`
+    @media ${device.tabletL} {
+        display: flex;
+        justify-content: center;
+    }
+`
+const StyledLinkText = styled(LinkText)`
+    @media ${device.tabletL} {
+        font-size: 12px;
+    }
+`
+const SignupAffiliate = ({
+    autofocus,
+    clearEmail,
+    email,
+    email_error_msg,
+    handleInputChange,
+    handleLogin,
+    handleValidation,
+    is_ppc,
+    is_submitting,
+}) => {
+    const [is_checked, setChecked] = useState(false)
+
+    const handleChange = (event) => {
+        setChecked(event.currentTarget.checked)
+    }
+
+    return (
+        <StyledFlex jc="flex-start" fd="column" p="4rem">
+            <Header as="h4" type="heading-3" mb="0.8rem">
+                {localize('Sign up')}
+            </Header>
+            <Header as="p" type="subtitle-1" weight="normal">
+                {localize('Enter your email address to begin')}
+            </Header>
+            {!is_ppc && (
+                <StyledNoteFlex fd="row" width="100%" p="8px 14px" mt="1.6rem">
+                    <div>
+                        <Header
+                            as="p"
+                            mb="0.2rem"
+                            weight="normal"
+                            notedBox
+                            color="grey-16"
+                            size="var(--text-size-xs)"
+                            tabletFontSize="12px"
+                        >
+                            {localize('Want to sign up as a trader?')}
+                            <LocalizedLinkText to="/signup">
+                                <StyledLinkText
+                                    id="dm-new-login-button"
+                                    ml="0.5rem"
+                                    size="14px"
+                                    color="grey"
+                                >
+                                    Create a Deriv account
+                                </StyledLinkText>
+                            </LocalizedLinkText>
+                        </Header>
+                    </div>
+                </StyledNoteFlex>
+            )}
+            <InputGroup>
+                <Input
+                    id="dm-email-input"
+                    name="email"
+                    type="text"
+                    border="solid 1px var(--color-grey-7)"
+                    labelColor="grey-5"
+                    labelHoverColor="grey-5"
```

Comment:
Please pull the latest upstream master. Some of these props have been renamed to snake_case instead of the camelCase here 🙏

Reply:
Refactored here, and checked other occurrences